docs/getting-started/install-configure.md

```diff
@@ -442,7 +442,7 @@ watch kubectl get pkg
 
 ```console
 # create service principal with Owner role
-az ad sp create-for-rbac --role Contributor --scopes /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx > "creds.json"
+az ad sp create-for-rbac --role Contributor --scopes --sdk-auth /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx > "creds.json"
```

Comment:
@bluedog13 thanks for the update here! I wonder if instead of adding back in the deprecated flag, we could use the flags described in https://github.com/crossplane/crossplane/issues/3261?

xref https://github.com/crossplane/crossplane/pull/3166

Reply:
ping @bluedog13 :)